junit-vintage-engine/src/test/java/org/junit/vintage/engine/JUnit4ParametrizedExample.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2015-2017 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v1.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.junit.vintage.engine;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+@RunWith(Parameterized.class)
+public class JUnit4ParametrizedExample {
```

Comment:
Please rename the class to `JUnit4ParameterizedTestCase` to align with our coding standards.